es-svc.yaml

```diff
@@ -10,7 +10,9 @@ spec:
   selector:
     component: elasticsearch
     role: client
+  clusterIP: 10.0.0.110
   ports:
   - name: http
+    nodePort: 30101
```

Comment:
This is for accessing the service from outside the cluster. That is outside the Kubernetes network. With nodePort you can hit the URL with your actual node IP address & node IP. Otherwise Kubenetes IPs are not accessible from the outside environment.